mono/metadata/threadpool-worker-default.c

```diff
@@ -221,6 +221,7 @@ rand_next (gpointer *handle, guint32 min, guint32 max)
 static void
 destroy (gpointer data)
 {
+#if 0
```

Comment:
What I mean is if someone is reviewing the code in a month they would not know why this occurred unless they used git blame, and even then they might not find the PR / its message